webapp/src/components/github_repo_selector/github_repo_selector.jsx

```diff
@@ -29,26 +30,18 @@ export default class GithubRepoSelector extends PureComponent {
     }
 
     componentDidMount() {
-        this.getRepos();
+        if (this.props.yourRepos) {
+            this.props.actions.getRepos();
```

Comment:
This doesn't make much sense to me. `props.yourRepos` should always be present already. Was there a case during development where this wasn't the case?

Reply:
I believe this was just left in from debugging and is not needed.  Removing